Plugzy.Infrastructure/Interface/Repository/IUserRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Plugzy.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Plugzy.Infrastructure.Interface.Repository
+{
+    public interface IUserRepository:IGenericRepositoryAsync<AppUser>
```

Comment:
her class için ayrı datarepo yapma,  tek bir DataRepository<T>   ile ilerleyebilirsin